enatega-multivendor-app/src/components/Sidebar/LogoutModal/styles.js

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from'react'
+import { StyleSheet } from 'react-native'
+import { alignment } from '../../../utils/alignment'
+import { theme } from '../../../utils/themeColors'
+import { scale } from '../../../utils/scaling'
+import ThemeContext from '../../../ui/ThemeContext/ThemeContext'
+
+const styles = (props = null) => {
+    const themeContext = useContext(ThemeContext)
```

Comment:
You are already passing current theme from LogoutModal file in styles. Either remove that or remove current theme in this file.